homeassistant/components/sonos/media_player.py

```diff
@@ -236,6 +236,7 @@ async def async_service_handle(service_call: ServiceCall) -> None:
             vol.Optional(ATTR_NIGHT_SOUND): cv.boolean,
             vol.Optional(ATTR_SPEECH_ENHANCE): cv.boolean,
             vol.Optional(ATTR_STATUS_LIGHT): cv.boolean,
+            vol.Optional(ATTR_EQ_BASS): vol.Coerce(int),
```

Comment:
You could take it a step further and restrict the range as a validator:
```suggestion
            vol.Optional(ATTR_EQ_BASS): vol.All(
                vol.Coerce(int), vol.Range(min=-10, max=10)
            )
```

Reply:
I think i already stumbled upon that above. Seems a better way. Thanks!